.github/workflows/publish_helm_charts.yml

```diff
@@ -0,0 +1,42 @@
+# this is WIP and needs a revision and testing before merging. Also we need to add this secrets in GH secrets
+name: Publish helm charts
+on:
+# Will only run when release is published. 
+  push:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  publish-helm-charts:
+    runs-on: ubuntu-20.04
+    container: hypertrace/helm-gcs-packager:0.3.0
+    # GITHUB_REF will be refs/tags/docker-MAJOR.MINOR.PATCH
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}       
+   
+      - name: package and release charts
+        env:
+          HELM_GCS_CREDENTIALS: ${{ secrets.HELM_GCS_CREDENTIALS }}
```

Comment:
Do these need to be in the env or can we reference them in run directly? As soon as they're available in env, we dramatically increase our attack surface area.

Reply:
It can be referenced directly in run as well.